packages/java-shell/package.json

```diff
@@ -12,7 +12,8 @@
   },
   "scripts": {
     "browserify-shell-api": "browserify src/main/js/all.js -o src/main/resources/js/all-standalone.js -dv",
-    "test-ci": "npm run test"
+    "test": "mocha -r \"../../scripts/import-expansions.js\" --timeout 60000 --colors -r ts-node/register \"./src/test/js/run-tests.ts\"",
```

Comment:
What do i need locally to run this? Should we add some docs?

Reply:
It’s the same as the other packages – it’s part of `npm test` on the root module. Whether that’s good or not is another question, it requires Java, but then again `npm test` already requires stuff like headless chrome, so … :woman_shrugging: